clang-tools-extra/clang-tidy/tool/clang-tidy-diff.py

```diff
@@ -374,6 +379,8 @@ def main():
         common_clang_tidy_args.append("-extra-arg-before=%s" % arg)
     for plugin in args.plugins:
         common_clang_tidy_args.append("-load=%s" % plugin)
+    if args.warnings_as_errors != "":
```

Comment:
probably this would be sufficient:
```suggestion
    if args.warnings_as_errors:
```